todocore/src/main/java/com/knighttodo/todocore/rest/request/DayTodoRequestDto.java

```diff
@@ -16,6 +16,9 @@
 @Builder
 public class DayTodoRequestDto {
 
+    @NotNull
+    private Integer orderNumber;
```

Comment:
I think that if this value can't be null we can make it primitive int value instead of boxing it into Integer.

Reply:
Done.